bootstrap-into-dart.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Bootstrap into Dart
+permalink: /bootstrap-into-dart/
+---
+
+New to the [Dart](https://www.dartlang.org) language?
+We've cherry-picked our favorite resources to
+help you quickly learn Dart.
+We [looked at a lot of languages](/faq/#why-did-flutter-choose-to-use-dart),
+and we found Dart easy and fun to learn, and we hope these resources make it
+easy for you to learn, too.
+
+[Language tour](https://www.dartlang.org/docs/dart-up-and-running/ch02.html)
+: Your best introduction to the Dart language.
+
+[Library tour](https://www.dartlang.org/docs/dart-up-and-running/ch03.html)
+: A good overview of Dart's powerful core libraries.
+
+[Effective Dart](https://www.dartlang.org/effective-dart/)
+: Guides for style, authoring documentation, usage, and more.
+
+[Futures Tutorial](https://www.dartlang.org/docs/tutorials/futures/)
+: Learn how to use Futures, a core class for one-time async events.
+
+[Streams Tutorial](https://www.dartlang.org/docs/tutorials/futures/)
```

Comment:
When would developers typically use Streams when building Flutter apps? It would be good to mention some example use cases for using specific Dart features in Flutter, just as a way to make things more concrete for readers.